### ISSUE TITLE
rubygems version issue

### ISSUE THREAD

**Issue by wdiechmann** [gjtorikian/commonmarker#263]
I know this is a long shot - and not commonmarker's "fault" - but perhaps you might have a trick up your sleeve 
🙏 

I'm using Dokku to deploy my Rails app from a Macbook Air M1. My local rubygems version is 3.4.22, my ruby version is 3.1.2p20 (newest available using `asdf list all ruby`) 

When I do `git push dokku` (which will deploy the app on a Ubuntu 20.04 LTS VM somewhere) the rubygems version used by the ruby-buildpack is 3.3.7 which is less than the 3.3.22 required by commonmarker-1.0.0.pre12-x86_64-linux.

I cannot find docs mentioning how to set the rubygems version - not on [Dokku](https://dokku.com), [Bundler](https://bundler.io), nor on [Heroku](https://devcenter.heroku.com/articles/buildpacks)

You guys are my last "straw" 
😢 

ps: I know - I can switch to Kramdown and be done with it - but...

**Comment by wdiechmann** [comment]
Thx for getting back to me 🤜A less ambitious req in rubygems would be much appreciated though 😉Med venlig hilsenWaltherDen 12. dec. 2023 kl. 14.24 skrev Garen Torikian ***@***.***>:﻿
I don’t know enough about Dokku to give more specific advice, but to update your Rubygems version you call gem update --system.

**Comment by gjtorikian** [comment]
I can’t; there’s a bug in older versions: https://github.com/gjtorikian/commonmarker/blob/404a9d20cd48905c05646b86bfd5cbc4d01bebf4/commonmarker.gemspec#L17

**Comment by wdiechmann** [comment]
I'll just add a quick comment in case someone else ends up here - 

I resolved my issue with a good measure of "system maintenance" - ie. I started to care about keeping my toolbelt in pristine condition aka:

```
brew upgrade ruby-build
asdf plugin update --all
```

this in turn allowed me to install newer versions of Ruby (3.2.2 in casu) which then allowed the Heroku buildpack-ruby to pick up a recent rubygems (3.3.26 I like to think) -- and suddenly Commonmarker v1.0.0 was all the rage!

My apologies @gjtorikian for waisting your bandwidth on a trivial system maintenance job all forgotten!
